src/utils/formatters.js

```diff
@@ -24,14 +24,13 @@ export function roundEth(eth: string): string {
  * @return {string} Formatted balance + successful suffix if balance defined,
  * corresponding string if not defined.
  */
-export function prettyWalletBalance(
```

Comment:
Why is that function changed? It seems like it don't work as it planned to work before. It should round balance and add suffix in case balance is present.

Reply:
Was simplified to the way now we need it with the balances get by the ethereumService

Reply:
Since we now store balance with 18 digits we need it to be reverted back again as far as I see. Furthermore, we use successful suffix property on some screens, I guess, but new implementation ignore that.